contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,32 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks of GitHub repositories aren't updated automatically. In order to keep your fork of the repository up-to-date with the latest changes and avoid merge conflicts, you should update it regularly.
+
+There are two ways to update your fork.
+
+1. Via the GitHub web interface.  Click `Fetch upstream` and then `Fetch and merge` on the fork:
+
+![Fetch and merge button in GitHub](https://user-images.githubusercontent.com/50295997/119873818-393e2000-bf25-11eb-8a0a-b804c293439b.png).
```

Comment:
I suppose this link is fine and will never break, but ideally we'd want to include this screenshot in the repository itself.  Perhaps in https://github.com/tldr-pages/tldr/tree/main/images?

Reply:
The link never dies, Dan use that in the README of the popular https://github.com/dandavison/delta and rocks.

Reply:
Yeah, that's a good point. 🤔

What about offline support then? 😉

Reply:
That convinced me, I'll include it in the repo.

Reply:
I can't really get this to work. I'm using the html `<img>` tag to insert the image, but if the image in `/images` it can't be found using `../images/image.png`. However if I put it in the root derictory it works with `../image.png`

Reply:
If I put it in the root derictery it works with `grip`, but not with `markdown-previewer`.

Reply:
@marchersimon shouldn't need a HTML `<img />` here - the Markdown syntax should work fine?

Reply:
It works with neither. Also, we use `<img>` for the banner in the README.